common/changes/office-ui-fabric-react/label-for-breadcrumb_2018-02-23-00-57.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "added More items aria label",
+      "type": "patch"
```

Comment:
this is technically speaking a minor bump (because we are adding a new prop, if you take dependency on this you are not backwards compatible with previous versions any longer).